simulator/src/utils.js

```
@@ -1,126 +1,137 @@
-import simulationArea from './simulationArea';
-import {
-    scheduleUpdate, play, updateCanvasSet, errorDetectedSet, errorDetectedGet,
-} from './engine';
-import { layoutModeGet } from './layoutMode';
-import plotArea from './plotArea';
+import { circuitProperty } from './circuit';
```

ESLint

[error] Dependency cycle detected.

(import/no-cycle)